controllers/admin/AdminThemesController.php

```diff
@@ -78,7 +78,7 @@ public function init()
 
     public function downloadAddonsThemes()
     {
-        if (!$this->logged_on_addons) {
+        if (!$this->logged_on_addons || (!in_array($this->authorizationLevel(), array(AdminControllerCore::LEVEL_ADD, AdminControllerCore::LEVEL_EDIT, AdminControllerCore::LEVEL_DELETE) )) || _PS_MODE_DEMO_) {
```

Comment:
Also ^^

Never call directly `AdminControllerCore`. There is an override system. Just call `AdminController`